showmore_dyn/showmore_dyn.css

```diff
@@ -0,0 +1,39 @@
+.limit-height {
+       max-height: 250px;
+       overflow: hidden;
+}
+
+.wall-item-body-toggle {
+    width: 100%;
+    text-align: center;
+    background-image: linear-gradient(rgba(0,0,0,0), #f6f6f6);
+    cursor: pointer;
+    height: 50px;
+    position: absolute;
+    top: auto;
+    bottom: 0;
+    z-index: 11;
+    left: 0;
+    display: none;
+}
+.wall-item-body-toggle-text {
+    width: 100%;
+    position: absolute;
+    top: auto;
+    bottom: 0;
+    z-index: 12;
+    left: 0;
+    text-shadow: 0 0 5px #f6f6f6;
+    font-weight: bold;
+}
+.wall-item-body {
+    position: relative;
+    width: 100%;
+}
+.wall-item-body-wrapper {
```

Comment:
Why do you need a wrapper around what is already a wrapper?

Reply:
This was the easiest way I could find to put the two overlay divs containing the gradient and the "show more" link at the correct position above the body div.



While the body div can be larger than the limit-height, the wrapper height is set to limit-height and the overlay divs can then just be placed at the bottom of it. Appending them to the body div wouldn't work.

Reply:
And: I preferred that they are shown as overlays above the body div such that the content fades out via the gradient, instead of a hard cut which could be achieved by just inserting the "show more" link after the body div.

Reply:
I'm pretty sure I can get the same display without the wrapper but I haven't taken the time to confirm my hunch yet. I'll approve your version if I can't do it timely.

Reply:
I finally took the time, removed all mentions of the wrapper, appended the Show more to the item content, and... it looked almost the same. The only difference is a small padding at the bottom because you defined the wrapper as `inline-block` for a reason that escapes me at the moment. Gradient looks the same over the content, button is active, I don't think we need to keep the wrapper.

Wrapper:

![friendica-showmre-dyn-wrapper](https://user-images.githubusercontent.com/925415/76827334-f1861980-67f4-11ea-9411-fad78d6bb333.png)

No wrapper:

![friendica-showmre-dyn-no-wrapper](https://user-images.githubusercontent.com/925415/76827333-f0ed8300-67f4-11ea-8557-527210b199a3.png)